.github/workflows/validate.yaml

```diff
@@ -41,7 +41,16 @@ jobs:
 
       - name: Check if everything is consistent
         working-directory: ./src/github.com/${{ github.repository }}
-        run: git diff --exit-code
+        run: |
+          if [ -n "$(git status --porcelain)" ]; then
+            echo '::debug::Running `git status`'
+            git -c color.status=always status
+            echo '::debug::Running `git diff`'
+            git -c color.ui=always diff
+            echo '::error::Not all generated files are commited. Run `make generated-files` and commit files.'
+            echo '::warning::`make generated-files` needs to be run on GOPATH due to https://github.com/knative/pkg/issues/1287'
+            exit 33
+          fi
```

Comment:
This produce the following output: https://github.com/openshift-knative/serverless-operator/pull/875/checks?check_run_id=2286742732#step:6:74